src/main/java/com/yugabyte/sample/apps/SqlInserts.java

```diff
@@ -154,8 +154,9 @@ private PreparedStatement getPreparedInsert() throws Exception {
     if (preparedInsert == null) {
       close(insConnection);
       insConnection = getPostgresConnection();
+      insConnection.createStatement().execute("set yb_enable_upsert_mode = true");
       preparedInsert = insConnection.prepareStatement(
-          String.format("INSERT INTO %s (k, v) VALUES (?, ?);", getTableName()));
+          String.format("INSERT INTO %s (k, v) VALUES (?, ?) on conflict(k) do update set v=excluded.v;", getTableName()));
```

Comment:
No need to modify the INSERT statement if we are setting the gflag `yb_enable_upsert_mode`?